middlewares/validators/validator.helper.js

```diff
@@ -137,6 +137,24 @@ function nameValidator(fieldLocation, fieldname, optional = true) {
     }
 }
 
+// untested
+/**
+ * Validates that field pronoun is ascii only.
+ * @param {"query" | "body" | "header" | "param"} fieldLocation the location where the field should be found 
+ * @param {string} fieldname name of the field that needs to be validated.
+ * @param {boolean} optional whether the field is optional or not.
+ */
+function pronounValidator(fieldLocation, fieldname, optional = true) {
+    const pronoun = setProperValidationChainBuilder(fieldLocation, fieldname, "invalid pronoun");
+    if (optional) {
+        return pronoun.optional({
+            checkFalsy: true
+        }).isAscii().withMessage("must contain only ascii characters");
+    } else {
+        return pronoun.exists().withMessage("pronoun must exist").isAscii().withMessage("must contain only ascii characters");
```

Comment:
Do we want to set up a list of pronouns that are acceptable, or is that too spicy?

Reply:
I was thinking about that and was torn on it. I think the solution lies in the front end to have people pick from a long list of suggestions we provide but still leave it open to alternatives. I think the nature of this warrants keeping it relatively open ended.